src/screens/Library/components/InstallModal/index.tsx

```diff
@@ -173,7 +173,10 @@ export default function InstallModal({ appName, backdropClick }: Props) {
   return (
     <span className="modalContainer">
       {gameInfo?.game?.title ? (
-        <div className="modal">
+        <div className="modal close-icon-container">
+          <SvgButton onClick={() => backdropClick()}>
+            <FontAwesomeIcon className={'close-icon'} icon={faXmark} />
```

Comment:
You can do `className="close-icon"` in the SvgButton instead, that way you can position the `button` tag (here you are moving the `svg` tag outside of the button).

When you do that, you'll probably won't need the `cursor: pointer` property since it's alredy a `button` tag and the cursor is pointer by default.